DragonFruit.Six.API/Dragon6Client.cs

```diff
@@ -70,7 +70,7 @@ public string AppId
         /// </summary>
         protected abstract TokenBase GetToken();
 
-        public T Perform<T>(UbiApiRequest requestData) where T : class
+        public T Perform<T>(UbiApiRequest requestData, CancellationToken token = default) where T : class
```

Comment:
because of this change it might be worth adding the cancellationtoken overload to _all_ extensions?

Reply:
to be resolved in #180